Api/repository/context/BaseContext.cs

```diff
@@ -0,0 +1,33 @@
+using Microsoft.EntityFrameworkCore;
+
+public class BaseContext : DbContext
```

Comment:
We are gonna use Dapper as communication with the DB as agreed in the Plan

Reply:
Done :D